### ISSUE TITLE
Jetty Database Configuration

### ISSUE THREAD

**Issue by Neuw84** [dnmilne/wikipediaminer#2]
When the Wikiminer war gets undeployed, or when there is an error the JettyDatabase stays locked. Mayve using ServletContextListener class we could unlock the Database before that.

**Comment by dnmilne** [comment]
Do you mean the Berkeley Database?

**Comment by Neuw84** [comment]
Yes, The Berkley Database. When the web application crashes for any reason if you try to deploy another war (for example with a fix), you need to restart the Web Application Server (in my case) as the database stays locked. Maybe using the ServletContextListener (when the application or servlet gets destroyed, undeployed....) the database can be unlocked before that. 
Is a very small improvement in the code but should be a few lines of code, will some research on this and post the results
